modules/network-observability-packet-drops.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-packet-drops_{context}"]
+= Working with packet drops
+Packet loss occurs when one or more packets of network flow data fail to reach their destination. You can track these drops by enabling the `spec.agent.ebpf.features` and `spec.agent.ebpf.privileged` specifications in the `FlowCollector`.
++
+[IMPORTANT]
+====
+Cpu and memory usage increases when this feature is enabled.
+====
+
+.Prerequisites
+* Access to an {product-title} cluster with version 4.13. 
+* Kernel supported by Red Hat Enterprise Linux (RHEL) 9.2.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. 
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A sample configuration is as follows:
++
+[id="network-observability-flowcollector-configuring-pkt-drop_{context}"]
+.Configure `FlowCollector` for packet drops
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: DIRECT
+  agent:
+    type: EBPF
+    ebpf:
+      features:
+       - PKT_DROP            <1>                         
```

Comment:
I think its now changed to `PacketsDrop`

Reply:
🤦‍♀️ is DNS also changed?

Reply:
I'm realizing now actually that I noticed this last week on Thursday because I see it written in my notebook to change PKT_DRP again but then I misplaced this jotted note so TY for keeping me straight here.

Reply:
yup look at [PR#394](https://github.com/netobserv/network-observability-operator/pull/394/files#diff-f56c4ef1268ddd69d4da318cd3ffc855de718f70c7de5de760fd18ef69f79aebR18) but seems like they have introduced a bug while renaming DNSTracking and missed an `r`

Reply:
No worries, happy to help!